.changeset/node-compat-v4.md

```diff
@@ -0,0 +1,8 @@
+---
+"wrangler": major
+"miniflare": major
```

Comment:
add "@cloudflare/vite-plugin": patch since we remove the legacy node_compat check there.

Reply:
May not be necessary since I think that vite-plugin has a dependency on wrangler?